smithy-openapi/src/test/java/software/amazon/smithy/openapi/fromsmithy/protocols/AwsRestJson1ProtocolTest.java

```diff
@@ -266,4 +266,53 @@ public OperationObject updateOperation(
             return OpenApiMapper.super.updateOperation(context, shape, operation, httpMethodName, path);
         }
     }
+
+    @Test
+    public void convertsErrorStructuresWithSameErrorCodeWithoutUsingOneOf() {
+        Model model = Model.assembler()
+                .addImport(getClass().getResource("error-code-collision-test.smithy"))
+                .discoverModels()
+                .assemble()
+                .unwrap();
+        OpenApiConfig config = new OpenApiConfig();
+        config.setService(ShapeId.from("example#Example"));
+        config.setOnErrorStatusConflict(OpenApiConfig.ErrorStatusConflictHandlingStrategy.USE_PROPERTIES);
+        ObjectNode result = OpenApiConverter.create()
+                .config(config)
+                .convertToNode(model);
+        InputStream openApiStream = getClass()
+                .getResourceAsStream("error-code-collision-test-use-properties.openapi.json");
+
+        if (openApiStream == null) {
+            throw new RuntimeException("OpenAPI model not found for test case: "
+                    + "error-code-collision-test-use-properties.openapi.json");
```

Comment:
I think it would be more idiomatic to assert that the `InputStream` is not null instead, such as:


```
    InputStream openApiStream = getClass()
        .getResourceAsStream("error-code-collision-test-use-oneof.openapi.json");

    Assertions.assertNotNull(openApiStream);
    Node expectedNode = Node.parse(IoUtils.toUtf8String(openApiStream));
    Node.assertEquals(result, expectedNode);
```

Or at least to keep the code consistent use `fail` as is done [above in this file](https://github.com/awslabs/smithy/blob/98990e88585b2f55e75012ef41dbc04d9e934dd3/smithy-openapi/src/test/java/software/amazon/smithy/openapi/fromsmithy/protocols/AwsRestJson1ProtocolTest.java#L60).

Reply:
runtime exception kept, following discussion.